env/development.env.template

```diff
@@ -0,0 +1,23 @@
+NODE_ENV=development
+USE_HTTPS=false
+AUTH_KEYS={"public":"S3cr3t"}
+GOOGLE_CLIENT_INFO={}
+AZURE_STORAGE_GEN_CNNSTR=example
+AZURE_STORAGE_OCR_CNNSTR=example
+WEBSCRAP_API_BASEURL=http://example.com
+WEBSCRAP_API_GETMPS=/mps
+WEBSCRAP_API_GETMPD=/mp
+WEBSCRAP_API_DECLS=/decls
+WEBSCRAP_API_KEY=example
+BLOB_CONTAINER_GENERAL=general
+BLOB_MPS_CACHE=mpscache
+LOG_TABLE_NAME=integresdev
+LOGUI_TABLE_NAME=integresdevui
+MONITOR_OCR_QUEUE=true
```

Comment:
MONITOR_OCR_QUEUE should be set on false when running project locally unless you are really debugging the queue monitoring